xcodeproj/internal/xcodeproj_rule.bzl

```
@@ -462,10 +462,10 @@ targets.
             # Don't create targets for resource bundles in BwB mode, but still
             # include their files if they aren't unfocused
             focused_targets_extra_files.append(
-                (xcode_target.label, xcode_target.inputs.resources),
+                (xcode_target.label, depset([r[0] for r in xcode_target.inputs.resources.to_list()])),
```

You should find a way to not use `.to_list()`. Also, this isn't correct, is it? It would be changing the owner of these files from whatever we set to `xcode_target.label`. Should this instead be something like this?
```suggestion
                xcode_target.inputs.resources,
```

And for that to work, the resources in the tuple would have to be a `depset()`.

---

> You should find a way to not use .to_list()

Yeah I was iffy about it that I had to make it to list and then to depset again. I agree on trying to find a way not to use it. 

> Also, this isn't correct, is it? It would be changing the owner of these files from whatever we set to `xcode_target.label`

Yes, you're right.